### PR TITLE
Updated linear activation function with arguments, returns and note.

### PR DESCRIPTION
**Pull request by lukasfolle**
This is my first commit.
Please, feel free to comment on this small change.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -277,6 +277,9 @@ def linear(x):
 
   Returns:
       The linear activation: `x`.
+        
+  # Note
```
Comment (dynamicwebpaige) [inline tensorflow/python/keras/activations.py]: Please follow the same style convention for `Returns:`.

```
Note:
      Often used as the last layer of regression networks.
```

**Review by dynamicwebpaige — CHANGES_REQUESTED** [review]

**Review by lukasfolle — COMMENTED** [review]
Should be ok now.

**Review by lukasfolle — COMMENTED** [review]
Should be correct now

**Review by ymodak — APPROVED** [review]

**Review by lukasfolle — COMMENTED** [review]
@dynamicwebpaige Fixed in 613a70f!

**Review by ymodak — APPROVED** [review]

**Pull request merged**